src/collection-preferences/content-display/__tests__/content-display.test.tsx

```diff
@@ -174,6 +174,107 @@ describe('Content Display preference', () => {
     });
   });
 
+  describe.each<boolean>([false, true])('Filtering (with i18n = %s)', withI18nProvider => {
+    it('filters options', () => {
+      const wrapper = renderContentDisplay(
+        withI18nProvider
+          ? undefined
+          : {
+              contentDisplayPreference: {
+                ...contentDisplayPreference,
+                i18nStrings: {
+                  columnFilteringPlaceholder: 'Filter columns',
+                  columnFilteringAriaLabel: 'Filter columns',
+                  columnFilteringClearFilterText: 'Clear filter',
+                  columnFilteringNoMatchText: 'No matches found',
+                  columnFilteringCountText: count => (count > 1 || count === 0 ? `${count} matches` : `${count} match`),
+                },
+              },
+            },
+        withI18nProvider
+      );
+      const filterInput = wrapper.findTextFilter();
+      expect(filterInput).not.toBeNull();
+      filterInput!.findInput().setInputValue('Item 1');
+
+      expect(filterInput?.findInput().findNativeInput().getElement()).toHaveAttribute('placeholder', 'Filter columns');
+      expect(filterInput?.findInput().findNativeInput().getElement()).toHaveAttribute('aria-label', 'Filter columns');
+      expect(filterInput?.findInput().findClearButton()?.getElement()).toHaveAccessibleName('Clear filter');
+
+      expect(filterInput!.findResultsCount().getElement()).toHaveTextContent('1 match');
+
+      const options = wrapper.findOptions();
+      expect(options).toHaveLength(1);
+      expect(options[0].findLabel().getElement()).toHaveTextContent('Item 1');
+    });
+
+    it('clears filter and shows all options', () => {
```

Comment:
let's remove tests that do not depend on withI18nProvider outside of the describe.each block